.ci/lint_r_code.R

```diff
@@ -0,0 +1,109 @@
+
+library(lintr)
+
+args <- commandArgs(
```

Comment:
TIL `commandArgs`. Always better to use base R and reduce dependencies if it's easy. :)